### ISSUE TITLE
Reviewer interests in registration form is not accessible

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5184]
The accessibility audit identified that the reviewer interests input does not have a label. I believe that this is a limitation of the jQuery UI autocomplete field, and that even remediating the label issue will not be enough to make it accessible. 

We have developed a WCAG-compliant autocomplete field in our UI Library, so we may need to pull that component out of the library and use it on the registration page.

**Comment by NateWr** [comment]
Fixed as part of work in https://github.com/pkp/pkp-lib/issues/5187. The autosuggest feature when defining reviewer interests during user registration was removed. This is still available when editing the profile.
